server/plugin.go

```diff
@@ -460,6 +490,24 @@ func (p *Plugin) syncUsers() {
 			for {
 				newUser, appErr = p.API.CreateUser(newMMUser)
 				if appErr != nil {
+					if appErr.Id == "app.user.save.email_exists.app_error" {
+						user, gErr := p.API.GetUserByEmail(msUser.Mail)
+						if user.RemoteId != nil && *user.RemoteId != "" && strings.HasPrefix(user.Username, "msteams_") {
+							p.API.LogDebug("Activating the inactive user", "Email", msUser.Mail)
+							if gErr != nil {
+								p.API.LogError("Unable to get the invactive user", "Email", msUser.Mail, "Error", gErr.Error())
+								break
+							}
+
+							if err := p.API.UpdateUserActive(user.Id, true); err != nil {
+								p.API.LogError("Unable to activate the user", "Email", msUser.Mail, "Error", err.Error())
+							}
+						} else {
+							p.API.LogError("Unable to create new user", "Email", msUser.Mail, "Error", gErr.Error())
```

Comment:
```suggestion
							p.API.LogError("Unable to create new user", "Email", msUser.Mail, "Error", appErr.Error())
```
Shouldn't it be like this?